MapboxNavigation/Style.swift

```diff
@@ -512,3 +514,51 @@ class ManeuverContainerView: UIView {
         }
     }
 }
+
+/// :nodoc:
+@objc(MBStatusView)
+class StatusView: UIView {
+    @IBOutlet weak var activityIndicatorView: UIActivityIndicatorView!
+    @IBOutlet weak var textLabel: UILabel!
+    @IBOutlet weak var topConstraint: NSLayoutConstraint!
+    
+    func show(_ title: String, showSpinner: Bool, duration: TimeInterval) {
+        textLabel.text = title
+        activityIndicatorView.isHidden = !showSpinner
+        activityIndicatorView.startAnimating()
+        
+        show()
```

Comment:
They're fileprivate so shouldn't affect too much, but if you have a better name 😊 
How about `updateConstraints(showOrHide: Bool)`? 🤔